components/user/ModalCollection.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <!-- TODO: move texts to translations -->
+  <b-modal
+    id="modal-collection"
+    hide-header
+    hide-footer
+  >
+    <div v-if="showForm">
+      <h2>Create new collection</h2>
+      <b-form @submit.stop.prevent="submitForm">
+        <b-form-group
+          label="Collection name"
+          label-for="collection"
+        >
+          <b-form-input
+            id="collection"
+            type="text"
+            required
+          />
+        </b-form-group>
+        <b-form-group
+          label="Collection description"
+          label-for="description"
+        >
+          <b-form-textarea
+            id="collection"
+          />
+        </b-form-group>
+        <div class="modal-footer">
+          <b-button
+            variant="outline-primary"
+            @click="goBack()"
+          >
+            Go back
+          </b-button>
+          <b-button
+            variant="primary"
+            type="submit"
+          >
```

Comment:
Shouldn't this be disabled before mandatory fields are entered?

Reply:
That is a good idea, but not clearly written in the acceptance criteria.